probe-rs/src/probe/mod.rs

```diff
@@ -76,6 +76,8 @@ pub enum DebugProbeError {
     Attached,
     #[error("Some functionality was not implemented yet")]
     NotImplemented(&'static str),
+    #[error("Some functionality was not implemented yet")]
+    AttachFailed(&'static str),
```

Comment:
The message looks wrong

Reply:
Removed the entire error as it is not neded anymore =)